.github/workflows/release.yml

```diff
@@ -0,0 +1,320 @@
+        run: |
+          pip install --pre --find-links ./wheels/ "scikit-decide[all]"
+          pytest -v -s tests/autocast
+          pytest -v -s tests/solvers/cpp
+          pytest -v -s tests/solvers/python
+          pytest -v -s tests/scheduling
+
+  upload:
+    needs: [test-unix, test-macos, test-windows]
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Download artifacts
+        uses: actions/download-artifact@v1.0.0
+        with:
+          name: wheels
+
+      - name: Get the version
+        id: get_version
+        run: |
+          echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
+
+      - name: Upload artifacts 📦 to release
+        uses: svenstaro/upload-release-action@v1-release
+        with:
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          file: wheels/*.whl
+          tag: ${{ steps.get_version.outputs.VERSION }}
+          overwrite: true
+          file_glob: true
+
+  deploy:
+    needs: [upload]
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Download artifact
+        uses: actions/download-artifact@v1.0.0
+        with:
+          name: wheels
+
+      - name: Publish distribution 📦 to PyPI
+        env:
+          PYPI_TOKEN: ${{ secrets.PYPI_PASSWORD }}
+        if: github.repository == 'airbus/scikit-decide' && env.PYPI_TOKEN != ''
+        uses: pypa/gh-action-pypi-publish@master
+        with:
+          password: ${{ secrets.PYPI_PASSWORD }}
+          packages_dir: wheels/
+
+      - name: Publish distribution 📦 to Test PyPI
+        env:
+          TEST_PYPI_TOKEN: ${{ secrets.TEST_PYPI_PASSWORD }}
+        if: env.TEST_PYPI_TOKEN != ''
+        uses: pypa/gh-action-pypi-publish@master
+        with:
+          password: ${{ secrets.TEST_PYPI_PASSWORD }}
+          packages_dir: wheels/
+          repository_url: https://test.pypi.org/legacy/
+
+  build-docs:
+    needs: [deploy]
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+
+      - name: Setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: Download artifacts
+        uses: actions/download-artifact@v1.0.0
+        with:
+          name: wheels
+
+      - name: Install scikit-decide
+        run: |
+          pip install --pre --find-links ./wheels "scikit-decide[all]"
+
+      - name: Generate
+        run: |
+          (cd docs; python autodoc.py)
+
+      - name: generate documentation
+        run: yarn global add vuepress && yarn install && yarn docs:build && touch docs/.vuepress/dist/.nojekyll
+
+      - name: Deploy 🚀
+        uses: JamesIves/github-pages-deploy-action@releases/v3
+        with:
+          ACCESS_TOKEN: ${{ secrets.gh_access_token }}
+          BRANCH: gh-pages # The branch the action should deploy to.
+          FOLDER: docs/.vuepress/dist # The folder the action should deploy.
+          GIT_CONFIG_EMAIL: guillaume.alleon@gmail.com
```

Comment:
@galleon To change to an Airbus email address as soon as we have one.

Reply:
Yes, you can always open an issue as a reminder.